packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -242,6 +244,43 @@ export const prepareStatusManager = (
         );
       },
 
+      /**
+       * If minted while advancing, publish a status update for the advance
+       * to vstorage.
+       *
+       * Does not add or amend `pendingSettleTxs` as this has
+       * already settled.
+       *
+       * @param {EvmHash} txHash
+       * @param {boolean} success whether the Transfer succeeded
+       */
+      notifyAdvanceOutcome(txHash, success) {
```

Comment:
This name is awkward since `advanceOutcome` is already "notifying".

I get that the existing one is keyed by (nfa, amount) and this is keyed by txHash. So the existing one is for "pending" transactions and this new one is for settled.

Line two of your comment draws the contrast being "settled". Let's just use that in the name. Something like,
```suggestion
      advanceOutcomeForSettled(txHash, success) {
```